misk-zookeeper/build.gradle

```diff
@@ -7,7 +7,9 @@ buildscript {
 apply plugin: 'kotlin-jpa'
 
 dependencies {
-  compile dep.curatorFramework
+  compile(dep.curatorFramework) {
```

Comment:
These `compile` declarations should be `api` (same thing, but newer).

I believe this does what you want: `dep.curatorFramework` should be made available to misk apps through this (transitive) dependency. Misk apps that use _shouldn't_ need to directly depend on `dep.curatorFramework` -- instead they can use the version supplied by Misk.

Reply:
If misk app _do_ depend on curatorFramework explicitly, they'll need to do the same exclusion. Explicitly depending on curatorFramework should be discouraged though.

Reply:
> If misk app do depend on curatorFramework explicitly, they'll need to do the same exclusion.

Ideally they would not need this exclusion. But I guess we can live with this.

Reply:
They can avoid depending on curatorFramework entirely though!

```gradle
// ZookeeperApp
dependencies {
  api dep.misk // <- has api.curatorFramework, so curatorFramework is available to this app for usage
  ...
}
```